Cargo.toml

```diff
@@ -41,7 +41,7 @@ tree-sitter = { version = "0.22" }
 nucleo = "0.2.0"
 
 [workspace.package]
-version = "23.10.0"
+version = "24.3.0"
```

Comment:
this should be 24.03.0 rather than 24.3.0?

relates to https://github.com/Homebrew/homebrew-core/pull/167564

Reply:
Ideally yes but Cargo doesn't consider it a valid version

Reply:
Is there any cargo issue for this?

Somehow this version scheme worked before? see https://github.com/Homebrew/homebrew-core/pull/131283

Reply:
I don't think so - it's a conflict of us using the version as CalVer but cargo explicitly only supporting SemVer so I doubt cargo would be interested in changing it.

The way we handle the version changed in the 23.10 release with https://github.com/helix-editor/helix/pull/8925 but this effect wasn't visible yet because of the October release (`10`). Previously we had all crate versions at v0.6.0 (the last SemVer release version) and used an ad-hoc `VERSION` file and constant for the CalVer version.